Makefile.dep

```diff
@@ -629,6 +629,20 @@ ifneq (,$(filter spiffs,$(USEMODULE)))
     USEMODULE += mtd
 endif
 
+ifneq (,$(filter fatfs_vfs,$(USEMODULE)))
+    USEPKG += fatfs
+    USEMODULE += vfs
+
+    ifeq ($(BOARD),native)
+      USEMODULE += fatfs_diskio_native
+      FATFS_DISKIO_NATIVE_DEFAULT_FILE ?= \"riot_fatfs_disk.img\"
+    else
+      USEMODULE += fatfs_diskio_sdcard_spi
+      USEMODULE += auto_init_storage
+    endif
```

Comment:
> Ok I agree on most of this. But consider a board that simply provides SPI. This board doesn't (and in my understanding shouldn't) know anything about a storage device. In one application the SPI may be used to connect to some drive (i.e. sdcard) in another application, using the same board, it may as well be possible that no storage/drive exists at all. If a board provides some kind of drive - yes then it should also provide/handle the information about it. This in turn leads me to the point that in case of native the information about the image file should indeed be placed in the boards Makefile because its a drive that native provides. Great we talked about it :)

Doesn't need to be done in this PR, but maybe we should use a similar mechanism we currently use for boards that connect network devices optionally (like e.g. an xbee adapter). In those the configuration comes from the application.